utilities.tex

```diff
@@ -296,10 +299,11 @@
 }}}}
 \end{codeblock}
 \exitexample
+} %% \color{oldclr}
 \end{addedblock}
 
-\ednote{Before [refwrap], insert the following section and renumber subsequent sections
-as appropriate. (Renumbering hasn't been performed herein to ease review.)}
+\ednote{\oldtxt{Before [refwrap], insert the following section and renumber subsequent sections
+as appropriate. (Renumbering hasn't been performed herein to ease review.)}}
```

Comment:
Ah, I took "strike" as ~~strikethrough~~, rather than actually removing it. Duly noted.
Which types of issue should I focus on for now?